exercises/practice/reverse-string/.meta/config.json

```diff
@@ -4,13 +4,13 @@
   ],
   "files": {
     "solution": [
-      "reverse_string.rb"
+      "reverser.rb"
```

Comment:
Generally speaking, though it is not always the case, the file name has matched the slug of the exercise  (with the exception of the `-` being `_` when representing two words.